v0/destinations/userlist/transform.js

```diff
@@ -0,0 +1,52 @@
+const _ = require("lodash");
+
+const { EventType } = require("../../../constants");
+const {
+  defaultRequestConfig,
+  defaultPostRequestConfig
+} = require("../../util");
+
+const { endpoint } = require("./config");
+
+function isBlank(value) {
```

Comment:
Please move this method to `v0/util/index.js`.

Reply:
✅